doc/user_guide/examples.rst

```diff
@@ -397,14 +398,33 @@ code that is output is the same as the original (but looks different
 as it has been translated to PSyIR and then output by the PSyIR
 fortran back-end).
 
-Example 16: Kernel Extraction
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The subdirectory ``full_example_extract`` contains a runnable example
-of LFRic code that creates a NetCDF file with the input and output
-parameters of a simple kernel. This example requires the installation
-of a Fortran compiler and NetCDF development environment. After compilation
-of the code, you can run the example. which will produce a NetCDF file:
+Example 17: Runnable Simplified LFRic Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This directory contains three simplified LFRic examples, that can be
+compiled and executed - a suitable Fortran compiler is of course
+required. The examples are using a subset of the LFRic infrastructure
+library, which is contained in PSyclone and which has been slightly
+modified to make it easier to create stand-alone, non-MPI LFRic codes.
```

Comment:
I don't think this should be part of a simple restructuring PR. I've opened #1158 for this (e.g. we need to check that indeed all makefiles are consistent, and at least one in ``lib/profiling`` is not).

Reply:
As outcome of the telco I have added a section on compilation to the examples, and updates all makefiles that still used FFLAGS. As far as grep tells me there is no more FFLAGS anywhere now. This might not be the full scope of #1158 (e.g. I don't describe how recompiling work) - you decide if this fixes #1158 or not :)